src/fsModules.ts

```diff
@@ -0,0 +1,49 @@
+import { lstatSync, readdirSync } from "fs";
+import { ChoiceType, Question } from "inquirer";
+import { join } from "path";
+import { nameRequired } from "./validator/nameRequired";
+
+const isDirectory = ({ path }: any) => lstatSync(path).isDirectory();
+
+const getDirectories = (baseDir: any) =>
+  readdirSync(baseDir)
+    .map((dirName: string) => ({ path: join(baseDir, dirName), name: dirName }))
+    .filter(isDirectory);
+
+function fsModules() {
+  return getDirectories("./src");
```

Comment:
src? widziałem gdzieś że ustawiałeś rootDir więc nie wiem czy to nie powinno być tutaj jakoś uwzględniane